src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/NotSubmittableNotification.java

```diff
@@ -0,0 +1,20 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+
+public class NotSubmittableNotification extends Notification {
+
+  /**
+   * Construct a notification that explains that the exercise cannot be submitted from the plugin.
+   */
+  public NotSubmittableNotification() {
+    super(
+        "A+",
+        "Cannot submit exercise",
+        "This exercise can only be submitted from the A+ web interface.",
```

Comment:
a UX matter (that might be too tedious to implement), but how hard would it be to add a hyperlink to that particular "web interface" submission page into the notification (planned to make a similar proposal for the "set token" dialog as well)?

Reply:
Both are very good ideas! It shouldn't be too difficult, I'll try adding them